nengo_viz/static/viz.css

```diff
@@ -1,3 +1,8 @@
 .graph {
-    background: #eeeeee;
-}
\ No newline at end of file
+    background: white;
+    border: 1px solid white;
```

Comment:
That's what the :hover is for.  You only need to know where the components are when you're manipulating them.  The intent is to reduce visual clutter.  (i.e. the visual clutter from seeing the outlines of every component all the time.  notice that this is true whether we have a grey background or a border -- they both introduce visual clutter)